README.rst

```diff
@@ -10,8 +10,11 @@ which focuses on advanced Markov chain Monte Carlo and variational fitting
 algorithms. Its flexibility and extensibility make it applicable to a
 large suite of problems.
 
-Check out the `getting started guide <docs/source/notebooks/getting_started.ipynb>`__!
-
+<<<<<<< HEAD
```

Comment:
These shouldn't show up and are the result of a merge-conflict.

Reply:
@twiecki  yes sorry did a mistake by forgetting to save the file locally. My next commit in this branch removes these lines